### ISSUE TITLE
Release TwistedWeb-0.7

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2330]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2330|
|Type|enhancement|
|Created|2006-12-25 01:25:29Z|

Yay

<details><summary>Searchable metadata</summary>

```
trac-id__2330 2330
type__enhancement enhancement
reporter__radix radix
priority__highest highest
milestone__Web_0_7 Web-0.7
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1167009929000000 1167009929000000
changetime__1168115758000000 1168115758000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix removed owner|
|-|-|

Please review at releases/twisted-web-0.7-2330

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @radix<br>@exarkun set status to `new`|
|-|-|

Looks good.  Merge

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [19209]) Merge releases/twisted-web-0.7.x-2330

Author: release automation, radix
Reviewer: exarkun
Fixes [#2330](https://github.com/twisted/twisted/issues/2330)

release shrapnel from release of twisted-web 0.7.0.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
